generators/entity-client/templates/angular/src/main/webapp/app/entities/_entity-management-dialog.component.ts

```diff
@@ -61,26 +61,32 @@ import { <%= uniqueRel.otherEntityAngularName%>Service } from '../<%= uniqueRel.
         }
     }
 }); _%>
+<%_ if ( fieldsContainInstant || fieldsContainZonedDateTime ) { _%>
+import { StringToMomentPipe } from '../../shared/util/string-to-moment-pipe';
+import { MomentToStringPipe } from '../../shared/util/moment-to-string-pipe';
+<%_ } _%>
 
 @Component({
     selector: '<%= jhiPrefixDashed %>-<%= entityFileName %>-dialog',
     templateUrl: './<%= entityFileName %>-dialog.component.html'
 })
 export class <%= entityAngularName %>DialogComponent implements OnInit {
 
-    <%= entityInstance %>: I<%= entityAngularName %>;
+    private _<%= entityInstance %>: I<%= entityAngularName %>;
```

Comment:
Dont use _ as the field is already private

Reply:
I prefixed it by an _ because I needed a setter for this property

Reply:
why is that needed? just use `set<%= entityAngularName %>` as method name

Reply:
I think it's a matter of taste, doing a setter (`set  <%= entityAngularName %>(x)`) allows to refer to the method as if it were an attribute